lib/ui/natives.dart

```diff
@@ -83,29 +83,6 @@ Future<developer.ServiceExtensionResponse> _getImpellerEnabled(
   }));
 }
 
-@pragma('vm:entry-point')
```

Comment:
You can't remove these from the regular UI build.

Reply:
maybe add a separate setup method for the 3d scene and stub it in the regular version?

Reply:
It's still in the regular UI build via https://github.com/flutter/engine/pull/38676/files#diff-59c10ede29c01c127b31d48f6ab7ffb2633dbe85b0831cab377cafc9e389156bR41

Reply:
ahh got it.